### ISSUE TITLE
pre-trained models

### ISSUE THREAD

**Issue by zxy110** [paul007pl/VRCNet#5]
Hi, I found that the loss did not drop during training, so I would like to ask if you can make the pre-trained model publicly available? Thank you very much!

**Comment by paul007pl** [comment]
Thanks for asking.  Unfortunately, many pretrianed models that are reported in our paper are lost.  
But I trained a VRCNet (2048pts) recently, which achieves better results than our report.
Here is the link:
https://www.dropbox.com/sh/8uixt8dbqfoh1bn/AADbmET-i73hzMEvsYVYhT_Za?dl=0
and you can give it a try~

**Comment by zxy110** [comment]
Thanks a lot!

**Comment by j-merz** [comment]
Is there any chance you could provide a model pretrained for 4096 points?
